src/main/java/ru/hogwarts/school/controller/StudentController.java

```diff
@@ -28,8 +27,8 @@ public Collection<Student> getAllStudents() {
         return studentService.getAllStudents();
     }
 
-    @GetMapping("/findByAgeBetween")
-    public Collection<Student> findByAgeBetween(@RequestParam int min, @RequestParam int max) {
+    @GetMapping(params = {"min", "max"})
+    public Collection<Student> findByAgeBetween(@RequestParam(required = false) int min, @RequestParam(required = false) int max) {
```

Comment:
```suggestion
    public Collection<Student> findByAgeBetween(@RequestParam int min, @RequestParam int max) {
```
зачем required = false?